src/Makefile

```diff
@@ -1,19 +1,19 @@
-UTF8LEX_LIBRARY_VERSION = 1.0.0
-UTF8LEX_MAJOR = $(word 1, $(subst ., ,$(UTF8LEX_LIBRARY_VERSION)))
+UTF8LEX_LIBRARY_VERSION ?= 1.0.0
+UTF8LEX_MAJOR ?= $(word 1, $(subst ., ,$(UTF8LEX_LIBRARY_VERSION)))
 
 #
 # Parameters for gcc compiling .c into .o files, .o files into exes, etc:
 #
 CC = gcc
-# !!! CFLAGS = -Werror -fPIC -O2
-CFLAGS = -Werror -fPIC -O2 --debug
+CFLAGS = -Werror -fPIC -O2
+# CFLAGS = -Werror -fPIC -O2 --debug
```

Comment:
Turn off debug by default.

TODO: allow --debug to be added to the CFLAGS from the commandline (make CFLAGS_EXTRA=--debug or similar).